ethereum/contracts/CappedMintableToken.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity >=0.4.22 <0.6.0;
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+
+contract CappedMintableToken is ERC20Mintable {
+    uint256 private cap;
+
+    constructor(uint256 _cap) public {
```

Comment:
`ERC20Capped` has already a param that accepts a `cap`. You can pass it to the parent contract via a specific syntax.

```
  constructor(string memory name, string memory symbol, uint256 cap)  
    ERC20Detailed(name, symbol, 18) 
    ERC20Capped(cap)
    public {}
```